iotedgehubdev/certutils.py

```diff
@@ -227,6 +227,101 @@ def get_cert_file_path(id_str, dir_path):
     def get_pfx_file_path(id_str, dir_path):
         return os.path.join(dir_path, id_str, 'cert', id_str + EC.PFX_SUFFIX)
 
+    def load_cert_from_file(self, id_str, cert_path, key_path, key_passphrase):
+        if id_str in list(self._cert_chain.keys()):
+            msg = 'Could not load cert from file. Certificate already in chain. ID: {0}'.format(id_str)
+            raise EdgeValueError(msg)
+        cert_dict = {}
+        with open(cert_path, 'r') as cert_file:
+            cert_content = cert_file.read()
+            cert_dict['cert'] = crypto.load_certificate(crypto.FILETYPE_PEM, cert_content)
```

Comment:
Do we also add try block, to avoid file not exists or pem is not valid?

Reply:
If error occurred when load trusted ca, the command should fail. So rely on the crypto library's exception instead of catch the exception and provide our own exception. We have no extra information to be added to the exception